cli/compose/loader/full-struct_test.go

```diff
@@ -552,1041 +551,3 @@ func secrets(workingDir string) map[string]types.SecretConfig {
-      source: /opt/data
-      target: /var/lib/mysql
-    - type: bind
-      source: /foo
-      target: /code
-    - type: bind
-      source: %s
-      target: /var/www/html
-    - type: bind
-      source: /bar/configs
-      target: /etc/configs/
-      read_only: true
-    - type: volume
-      source: datavolume
-      target: /var/lib/mysql
-    - type: bind
-      source: %s
-      target: /opt
-      consistency: cached
-    - type: tmpfs
-      target: /opt
-      tmpfs:
-        size: 10000
-    - type: cluster
-      source: group:mygroup
-      target: /srv
-    working_dir: /code
-    x-bar: baz
-    x-foo: bar
-networks:
-  external-network:
-    name: external-network
-    external: true
-  other-external-network:
-    name: my-cool-network
-    external: true
-    x-bar: baz
-    x-foo: bar
-  other-network:
-    driver: overlay
-    driver_opts:
-      baz: "1"
-      foo: bar
-    ipam:
-      driver: overlay
-      config:
-      - subnet: 172.16.238.0/24
-      - subnet: 2001:3984:3989::/64
-    labels:
-      foo: bar
-  some-network: {}
-volumes:
-  another-volume:
-    name: user_specified_name
-    driver: vsphere
-    driver_opts:
-      baz: "1"
-      foo: bar
-  cluster-volume:
-    driver: my-csi-driver
-    x-cluster-spec:
-      group: mygroup
-      access_mode:
-        scope: single
-        sharing: none
-        block_volume: {}
-      accessibility_requirements:
-        requisite:
-        - segments:
-            region: R1
-            zone: Z1
-        - segments:
-            region: R1
-            zone: Z2
-        preferred:
-        - segments:
-            region: R1
-            zone: Z1
-      capacity_range:
-        required_bytes: "1073741824"
-        limit_bytes: "8589934592"
-      secrets:
-      - key: mycsisecret
-        secret: secret1
-      - key: mycsisecret2
-        secret: secret4
-      availability: active
-  external-volume:
-    name: external-volume
-    external: true
-  external-volume3:
-    name: this-is-volume3
-    external: true
-    x-bar: baz
-    x-foo: bar
-  other-external-volume:
-    name: my-cool-volume
-    external: true
-  other-volume:
-    driver: flocker
-    driver_opts:
-      baz: "1"
-      foo: bar
-    labels:
-      foo: bar
-  some-volume: {}
-secrets:
-  secret1:
-    file: %s/secret_data
-    labels:
-      foo: bar
-  secret2:
-    name: my_secret
-    external: true
-  secret3:
-    name: secret3
-    external: true
-  secret4:
-    name: bar
-    file: %s
-    x-bar: baz
-    x-foo: bar
-configs:
-  config1:
-    file: %s/config_data
-    labels:
-      foo: bar
-  config2:
-    name: my_config
-    external: true
-  config3:
-    name: config3
-    external: true
-  config4:
-    name: foo
-    file: %s
-    x-bar: baz
-    x-foo: bar
-x-bar: baz
-x-foo: bar
-x-nested:
-  bar: baz
-  foo: bar
-`,
-		filepath.Join(workingDir, "static"),
-		filepath.Join(workingDir, "opt"),
-		workingDir,
-		workingDir,
-		workingDir,
-		workingDir)
```

Comment:
Right; perhaps this was for running the tests on Windows? 🤔

Reply:
Looks to be indeed; https://github.com/docker/cli/commit/0cf2e6353a88a12b78da72db6a7e7c7d049d3ed8 (https://github.com/docker/cli/pull/905)

However, I don't think any of the code is converting these paths; these paths are also "daemon-side", so no conversion is done; there's also other paths in the same file that are unix-style (forward slashes), so I think we should be fine here.

Would be good perhaps if someone could double-check the unit-tests in this package on a Windows machine.